appveyor.yml

```diff
@@ -28,6 +28,7 @@ build_script:
   - C:\Ruby21\DevKit\devkitvars.bat
   - echo %PATH%
   - bundle exec rake build
+  - bundle exec rake build64
```

Comment:
Oh blargh, you're right.  I just removed the platform specific flag after adding the other ruby_versions to the matrix because autodetection seems to work like magic.